jfrog_platform_self-hosted/manifest.json

```
@@ -0,0 +1,81 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "b2748652-b976-461c-91dd-5abd4467f361",
+  "app_id": "jfrog-platform",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "View and analyze JFrog Artifactory and Xray Logs, Violations and Metrics",
+    "title": "JFrog Platform (Self-hosted)",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "JFrog Artifactory Logs dashboard",
+        "image_url": "images/jfrog_artifactory_logs.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Logo",
+        "image_url": "images/JFrog+Logo.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Artifactory Metrics dashboard",
+        "image_url": "images/jfrog_artifactory_metrics.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Xray Logs dashboard",
+        "image_url": "images/jfrog_xray_logs.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Xray Violations dashboard",
+        "image_url": "images/jfrog_xray_violations.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Xray Metrics dashboard",
+        "image_url": "images/jfrog_xray_metrics.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Containers",
+      "Category::Log Collection",
+      "Category::Metrics",
+      "Category::Security",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```

```suggestion
      "Supported OS::macOS",
      "Offering::Integration"
```
Please also add submitted data types in the following format:
`"Submitted Data Type::Metrics"`